packages/block-editor/src/components/block-draggable/index.js

```diff
@@ -102,7 +161,12 @@ const BlockDraggable = ( {
 				}
 			} }
 			__experimentalDragComponent={
-				<BlockDraggableChip count={ clientIds.length } icon={ icon } />
+				<BlockDraggableChip
+					count={ clientIds.length }
+					icon={ icon }
+					clientIds={ clientIds }
+					targetClientId={ targetClientId }
```

Comment:
Is this addition currently doing anything? It looks like `BlockDraggableChip` isn't currently using it?

Reply:
yeah I must have left that in while experimenting 😄